test/test-install.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+var exec = require('child_process').exec;
+var fmt = require('util').format;
+var install = require('../bin/sl-nginx-ctl-install');
+var path = require('path');
+var tap = require('tap');
+
+var user = 'nobody';
+var group = 'nobody';
+
+tap.test('setup', function(t) {
+  t.plan(2);
```

Comment:
Because `test/test-install.js` is the first tests run, this plan being the first line of TAP output confuses CI.

To fix it, it's probably easiest to split `setup` into 2 separate tests:
```js
tap.test('user id', function(t) {
  exec('id -un', function(err, stdout) {
    t.ifError(err, 'getting current user');
    user = stdout.trim();
    t.end();
  });
});

tap.test('group id', function(t) {
  exec('id -gn', function(err, stdout) {
    t.ifError(err, 'getting current group');
    group = stdout.trim();
    t.end();
  });
});
```